supertokens_python/recipe/emailverification/asyncio/__init__.py

```diff
@@ -85,7 +86,7 @@ async def revoke_email_verification_token(
     user_id: str,
```

Comment:
rename this to `def revoke_email_verification_tokens(`, and remove the one below from this file.

Reply:
done